lib/sync/index.js

```diff
@@ -17,26 +18,31 @@ function toJSON(dataset_id, returnData, cb) {
   return cb(null, {});
 }
 
+// Invoke the Sync Server.
 function invoke(dataset_id, params, callback) {
   syncUtil.doLog(dataset_id, 'info', 'invoke');
 
+  if (arguments.length < 3) throw new Error('invoke requires 3 arguments');
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reformat the code to have only one statement per line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AOneStatementPerLine)